.pylintrc

```diff
@@ -0,0 +1,32 @@
+[BASIC]
+include-naming-hint=yes
+method-rgx=[A-Z_][A-Za-z0-9]{2,49}$
+function-rgx=[A-Z_][A-Za-z0-9]{2,49}$|main|unicode_escape_decode
+const-rgx=[a-zA-Z_][A-Za-z0-9_]{2,49}$
+variable-rgx=[a-z_][a-z0-9_]{0,49}$
+argument-rgx=[a-z_][a-z0-9_]{0,49}$
+class-rgx=[A-Z_][a-zA-Z0-9]+$|basestring|unicode|long|xrange
+
+[MESSAGES CONTROL]
+disable=global-statement,multiple-statements,missing-docstring,no-else-return,no-self-use,consider-merging-isinstance,bad-indentation,bad-continuation,fixme,bad-option-value,anomalous-unicode-escape-in-string,unused-argument,useless-object-inheritance,consider-using-dict-comprehension,consider-using-in
```

Comment:
actually, bad_indentation should not be disabled, rather fix those lines. These lines are all lines added via our patches, so fixing those line will not affect our ability to follow upstream.

Reply:
I created PR #52 for that.